atef/bin/check.py

```diff
@@ -42,14 +39,15 @@ def build_arg_parser(argparser=None):
         "-v", "--verbose",
         action="count",
         help="Increase output verbosity",
+        default=0,
```

Comment:
Out of scope for this PR: is it simple for us to define common arguments like `verbose` in a more central place? Is that what the `argparser` argument is for here?

Reply:
It could be moved to `atef.bin.main` pretty easily. The only annoying thing might be that the verbosity level comes before the command instead of after:
`$ atef -v check ...`